articles/language/type-model.md

```diff
@@ -190,26 +170,22 @@ Function types are completely specified by their signature.
 For example, a function that computes the sine of an angle
 would have type `(Double -> Double)`.
 
-Operations -- but not functions -- may allow the application of
-one or more functors.
+Operations -- but not functions -- have certain additional characteristics that are expressed as part of the operation type. 
```

Comment:
Since this sentence implicitly introduces a new definition, I'd suggest typesetting the word being defined in italics.
I've also added a suggestion changing the parenthetical note below from UK style (en dashes "–" surrounded by spaces) to US style (em dashes "—" with no spaces).
```suggestion
Operations—but not functions—have certain additional _characteristics_ that are expressed as part of the operation type. 
```